### ISSUE TITLE
best pretrained model for cars

### ISSUE THREAD

**Issue by emjay73** [baegwangbin/surface_normal_uncertainty#13]
Thank you for sharing your code!

I'm quite new to the normal estimation area and wonder what is the best pre-trained model for cars and what is the coordinate that defines normal.

Which one is the correct one?

![image](https://github.com/baegwangbin/surface_normal_uncertainty/assets/36818801/465b4c33-4cfd-4fa4-a4cf-5b420d053b90)

Thank you.

**Comment by emjay73** [comment]
I figured it out by myself. I was able to obtain the right coordinates where the normals are defined by putting a minus sign to all the axes in OpenCV coordinates.

**Comment by Howie-Ye** [comment]
Actually, the coordinate is not OpenCV or OpenGL. I did some experiments and found that it's similar to OpenGL format with x-axis flipped. So the normal will be converted to OpenGL format by multiplying [-1,1,1] and to OpenCV format by multiplying [-1,-1,-1] .
